api/user/schema.py

```diff
@@ -212,11 +216,11 @@ def mutate(self, info, **kwargs):
         location_id = kwargs['location_id']
         query_user = User.get_query(info)
         user = query_user.filter(
-          UserModel.state == "active", UserModel.email == email).first()
+            UserModel.state == "active", UserModel.email == email).first()
         if not user:
             raise GraphQLError("User not found")
         new_location = LocationModel.query.filter_by(
-          id=location_id, state="active").first()
+            id=location_id, state="active").first()
```

Comment:
pay attention to changes u introduce on a file. eg adding extra white spaces on a file is not necessary